clientv3/watch.go

```diff
@@ -61,6 +61,11 @@ func (wr *WatchResponse) Err() error {
 	return nil
 }
 
+// IsProgressNotify returns true if the WatchResponse is progress notification.
+func (wr *WatchResponse) IsProgressNotify() bool {
+	return len(wr.Events) == 0 && !wr.Canceled
```

Comment:
> Canceled is mostly there to know whether the watcher lost the stream / couldn't establish the stream because of compaction.

I didn't get it. Does it mean the watcher lost the stream? Or couldn't establish the stream?